server/build_event_protocol/build_event_server/build_event_server.go

```diff
@@ -159,15 +159,12 @@ func (s *BuildEventProtocolServer) PublishBuildToolEventStream(stream pepb.Publi
 				streamID = in.OrderedBuildEvent.StreamId
 				ctx = log.EnrichContext(ctx, log.InvocationIDKey, streamID.InvocationId)
 				channel = s.env.GetBuildEventHandler().OpenChannel(ctx, streamID.InvocationId)
+				log.CtxInfof(ctx, "Opened a channel for stream ID %s.", streamID)
```

Comment:
nit: I think streamID is a struct - since the format string has `%s` idk if you're expecting it to be a string?

The context is enriched with the invocation ID so something like this might be fine:

```suggestion
				log.CtxInfo(ctx, "Opened invocation channel")
```